zhaquirks/tuya/__init__.py

```diff
@@ -844,6 +846,33 @@ class TuyaZBElectricalMeasurement(CustomCluster, ElectricalMeasurement):
     _CONSTANT_ATTRIBUTES = {AC_CURRENT_MULTIPLIER: 1, AC_CURRENT_DIVISOR: 1000}
 
 
+# Tuya Zigbee Cluster 0xE000 Implementation
+class TuyaZBE000Cluster(CustomCluster):
+    """Tuya manufacturer specific cluster 57344."""
+
+    name = "Tuya Manufacturer Specific"
+    cluster_id = TUYA_CLUSTER_E000_ID
+    ep_attribute = "tuya_is_pita_0"
```

Comment:
just curious... what is pita?

Reply:
Alexei was thinking it was on good name for one unknown tuya cluster but we have find fictions on one of them and implanting it here.
**P**ain
**I**n
**T**he
**A** . . (that you is have in side your pants on the back side)

Reply:
LMFAO! OK just wanted to confirm... Totally makes sense now! 😄